client/clickLogger.js

```diff
@@ -0,0 +1,36 @@
+var currQuery = "example";
+var numOfEntries = 0;
+const clicksBuffSize = 5;
+var clicksBuffer = new Array(clicksBuffSize);
+
+
+function setCurrQuery(query) {
+    currQuery = query;
+}
+
+function recordClick(redirect) {
+    // Logging clicks
+    var datetime = (new Date()).toString();
+    datetime = datetime.concat("\"", currQuery.toString(), "\" - ", redirect.toString(), "\n");
+    console.log("debug", `datetime: ${datetime}`);
+    if (numOfEntries < clicksBuffSize) {
+      clicksBuffer.push(datetime);
+      numOfEntries++;
+    } else {
+      var fs = require('fs')
```

Comment:
move import to top of file. Its a waste of server processing speed to reimport every time this function is run